src/Qowaiv.CodeAnalysis.CSharp/Qowaiv.CodeAnalysis.CSharp.csproj

```diff
@@ -20,25 +20,27 @@
     <PackageVersion>0.0.6.1</PackageVersion>
     <PackageIcon>package-icon.png</PackageIcon>
     <PackageReleaseNotes>
+ToBeReleased
+- QW0008: Value Types with an empty state should be used as nullable properties. #18
```

Comment:
I think there is a negation missing. "Value Types with an empty state should *not* be used as nullable properties."